content/blog/2023-10-27.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "2023-10-27"
+date = 2023-10-27
++++
+
+---
+
+Today is Friday, day 300 of 365 (82.19%). We're in week 43 of 52 (82.69%). 
+
+## Planning
+
+What I am going to do today: 
+
+- [ ] [learn QR Code capture with Flutter](https://github.com/orgs/OmnicodeSolutions/projects/3/views/1?pane=issue&itemId=41028440)
+
+## Activities executed
+
+(update): Today I did this and that
```

Comment:
Please mention we took a look at https://flutterawesome.com/fintech-dashboard-clone-built-with-flutter/ and discussed briefly about the theme of the OmniQRCodes app.